### ISSUE TITLE
Lack of support for accents and specific letters

### ISSUE THREAD

**Issue by Felipefpl** [builtbybel/CleanmgrPlus#25]
letters like "Ç" or letters with accents such as "á", "ê" or "ã" arent displayed properly by the program, the pic below shows how they are displayed in the program:

![Weird characters](https://user-images.githubusercontent.com/5569804/127714644-2d0e9526-8aa8-4cdc-ad3b-601fdc6f1290.gif)

**Comment by Belim** [comment]
Yeah, its the lack of unicode controls support. Maybe i will think about it again but currently it is too much time consuming
